setup.py

```diff
@@ -2,43 +2,57 @@
 from __future__ import division
 from __future__ import print_function
 from __future__ import absolute_import
+from __future__ import unicode_literals
```

Comment:
Can't import unicode_literals in setup.py. Had lots of issues with this in past. Prevents install on Py2 in certain conditions.